Lib/xml/dom/minidom.py

```diff
@@ -172,11 +184,16 @@ def removeChild(self, oldChild):
         oldChild.nextSibling = oldChild.previousSibling = None
         if oldChild.nodeType in _nodeTypes_with_children:
             _clear_id_cache(self)
-
         oldChild.parentNode = None
         return oldChild
 
     def normalize(self):
+        """Transform a node into its normalized form.
+
+        Removes empty exclusive Text nodes and concatenates the data of
```

Comment:
Please use the imperative mode for reference text (docstrings count as reference).

Reply:
Addressed in a4840f33d3d3c3292663a53e3a205feb1b8e3210